index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html> 
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Events</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/styles.css"/>
+    <link rel="stylesheet" href="node_modules/font-awesome/css/font-awesome.min.css">
+    <link href="images/favicon.png" rel="icon">
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
+</head>
+<body>
+    <nav class="navTop">
+        <header>
+            <div class="left">
+                <a href="#" alt="Events">
+                    <img class="" src="images/logo.png">
+                </a>
+            </div>
+            <ul class="right">                    
+                <li><a href="#">NOSOTROS</a></li>
+                <li><a href="#">PROMOCIONES</a></li>
+                <li><a href="#">MENÚ</a></li>
+                <li><a href="#">Haz tu Reservación</a></li>
+                <button type="button" name="button">CONTACTO</button>
+            </ul>
+        </header>
+    </nav>
+    <div class="segment-one">
+        <div class="content">
+            <div><h1>DELICIOSAMENTE</h1></div> 
+            <div><h1>DIVERTIDO</h1></div>
+            <span>RESTAURANTE + JUEGOS DE MESA</span>    
+        <div> <button type="button" name="button">HAZ UNA RESERVACIÓN</button></div>
+        </div>
+    </div>
+    <div class="discount">
+        <div class="content">
+            <span>¡Obtén tu cupón de DESCUENTO!</span>
+            <input type="Nombre" placeholder="Nombre*" name="name" id="">
+            <input type="email" placeholder="Email*" name="email" id="">
+            <button type="button" name="button">ENVIAR</button>
+        </div>
+    </div>
+    <div class="segment-two">
+        <div class="content">
+            <h2>NOSOTROS</h2> 
+            <p>El mejor Restaurante dedicado al espacio lúdico de Juegos de Mesa. Contamos con los mejores juegos de mesa, tienda, expertos en diversión y un menú deliciosamente divertido. 
+            <br><br>El mejor Restaurante dedicado al espacio lúdico de Juegos de Mesa. Contamos con los mejores juegos de mesa, tienda. expertos en diversión y un menú deliciosamente divertido. 
```

Comment:
Do not use `br`. In this case is better to have two paragraphs and margin between them.